_data/simple-icons.json

```diff
@@ -7599,7 +7603,7 @@
         {
             "title": "Redux",
             "hex": "764ABC",
-            "source": "https://github.com/reactjs/redux/tree/master/logo"
+            "source": "https://github.com/reduxjs/redux/blob/master/logo/logo.svg"
```

Comment:
Should be https://github.com/reduxjs/redux/blob/abb5f892f7e46849e5ca9bc13e75cfbc944a71b6/logo/logo.svg